### DISCUSSION TITLE
registerShape 方法是否被删除了？v5如何获取标记的画布坐标

### DISCUSSION THREAD

**Discussion by ChristmasFox** [antvis/G2#5072] [Q&A] [answered]
我们需要在v5中也能够自定义图形，v5中是否有类似draw(cfg: ShapeInfo, container: IGroup)这个方法能够使我获取相应标记的坐标点，来绘制自定义的图表。

**Comment by pearmini** [comment, marked as answer]
`registerShape` 对应的能力并没有移除，只不过这个 API 还在实验阶段，预计这两周暴露给用户。

**Reply by ChristmasFox** [reply]
Nice！

**Reply by ChristmasFox** [reply]
下周这个方法能暴露出来吗？
